### PR TITLE
master

### PR DESCRIPTION
**Pull request by tobidelbruck**
add / move papers, improve Contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1587,9 +1584,15 @@ IEEE Trans. Image Process., 2017, pp. 2192-2202. [PDF](https://www.neuromorphic-
 - <a name="Lee19arxiv"></a>Lee, S., Kim, H., Kim, H.J.,  
 *[Edge Detection for Event Cameras using Intra-pixel-area Events](https://bmvc2019.org/wp-content/uploads/papers/1140-paper.pdf)*,  
 British Machine Vision Conf. (BMVC), 2019. [PDF](https://arxiv.org/pdf/1907.07469)
+- <a name="liu19ieeesigproc"></a> S. Liu, B. Rueckauer, E. Ceolini, A. Huber, and T. Delbruck,
```
Comment (guillermogb) [inline README.md:1587]: This one is duplicated in the section "Neuromorphic Systems". Do you want to move it or do you want to have it in two sections, with one of them being a link to the other one?
See the example Bardow et al. CVPR 2016: it has a main entry and other entries linking to the main one.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Comment by tobidelbruck** [comment]
HI Guillermo, thanks for noticing. I will make link from one to the 
other. That paper should really be in signal processing section since 
that is the focus of it. I think we can take it out of the Neuromorphic 
Systems section entirely.  How should I do this?  Kill the pull request 
and make another one? I'm not familiar with using pull requests.

Thanks

**Comment by guillermogb** [comment]
Hi Tobi. You can still commit changes to this PR, can't you? Then we merge.
Alternatively, we can merge this and then modify it; it is not as good, but it can be done.
If you want to proceed in a different way, let me know.

**Comment by guillermogb** [comment]
Let me do it, to move forward with this.

**Review by guillermogb — APPROVED** [review]

**Pull request merged**